packages/libraries/router/src/registry_logger.rs

```diff
@@ -0,0 +1,98 @@
+use std::{env, fmt};
+
+static LOG_LEVEL_NAMES: [&str; 5] = ["ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
```

Comment:
maybe we can remove this bc it isn't used anywhere except in `Level::from_str`, and it just creates this extra jump to reveal the implementation of the method.

Reply:
we need it for `as_str`

Reply:
just a suggestion, both works perfectly!